fiat-roles/src/main/java/com/netflix/spinnaker/fiat/roles/UserRolesSyncer.java

```diff
@@ -201,6 +201,41 @@ public long syncAndReturn(List<String> roles) {
     }
   }
 
+  public long syncServiceAccounts(String serviceAccountId, List<String> roles) {
```

Comment:
If this is syncing one service account can we remove the plural from the method name? :)

Reply:
Absolutely, thanks.